tests/python/PythonPackageInitBearTest.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+import os
+
+from queue import Queue
+
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+
+from bears.python.PythonPackageInitBear import PythonPackageInitBear
+
+
+class PythonPackageInitBearTest(unittest.TestCase):
+
+    def setUp(self):
+        try:
+            os.mkdir('with-init')
+        except:
+            raise
+        open('with-init/__init__.py', 'a').close()
+
+        os.mkdir('ignored-no-init')
+        open('ignored-no-init/ignored.py', 'a').close()
+
+        os.mkdir('not-ignored-no-init')
+        open('not-ignored-no-init/test-file.py', 'a').close()
+
+        self.file_dict = {}
+        self.section = Section('PythonPackageInit')
+        self.queue = Queue()
+        Setting('ignored_dirs', 'ignored-no-init')
```

Comment:
I don't know what's you're trying to do here. Proper usage of the construct I think you're trying to use would be:
````python
x = Section("mysection")
x.add_or_create_setting("setting, "Key")
````